tests/upgrades/test_client.py

```diff
@@ -27,373 +28,311 @@
 from upgrade_tests.helpers.scenarios import dockerize
 from upgrade_tests.helpers.scenarios import get_entity_data
 
-from robottelo.api.utils import attach_custom_product_subscription
-from robottelo.api.utils import call_entity_method_with_timeout
-from robottelo.config import settings
+from robottelo.api.utils import enable_rhrepo_and_fetchid
+from robottelo.constants import DEFAULT_ARCHITECTURE
+from robottelo.constants import DEFAULT_CV
 from robottelo.constants import DEFAULT_ORG
-from robottelo.test import APITestCase
+from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME
+from robottelo.constants import ENVIRONMENT
+from robottelo.constants import FAKE_0_CUSTOM_PACKAGE_NAME
+from robottelo.constants import PRDS
+from robottelo.constants import REPOS
+from robottelo.constants import REPOSET
+from robottelo.constants.repos import FAKE_0_YUM_REPO
+from robottelo.constants.repos import FAKE_9_YUM_REPO
+from robottelo.test import settings
+
+
+# host machine for containers
+docker_vm = settings.upgrade.docker_vm
+SATHOST = settings.server.hostname
+
+
+@pytest.fixture(scope='module')
```

Comment:
https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/api_fixtures.py#L49

This fixture already exists, no need to re-define it here.

Reply:
OK, will do, but there are a few other problems. For example, the asserts like this:
`assert FAKE_0_CUSTOM_PACKAGE_NAME in installed_package`

are useless, because format is like so:

`>>> installed_package                                                                                                                                
'package gofer is not installed'`

Reply:
That is fixed now